docs/reference/docs/get.asciidoc

```diff
@@ -96,25 +95,23 @@ curl -XGET 'http://localhost:9200/twitter/tweet/1?_source=*.id,retweeted'
 
 
 [float]
-[[get-fields]]
-=== Fields
+[[get-stored-fields]]
+=== Stored Fields
 
 The get operation allows specifying a set of stored fields that will be
-returned by passing the `fields` parameter. For example:
+returned by passing the `stored_fields` parameter. For example:
 
 [source,js]
 --------------------------------------------------
-curl -XGET 'http://localhost:9200/twitter/tweet/1?fields=title,content'
+curl -XGET 'http://localhost:9200/twitter/tweet/1?stored_fields=title,content'
 --------------------------------------------------
 
-For backward compatibility, if the requested fields are not stored, they will be fetched
-from the `_source` (parsed and extracted). This functionality has been replaced by the
-<<get-source-filtering,source filtering>> parameter.
+If the requested fields are not stored, they will be ignored.
 
 Field values fetched from the document it self are always returned as an array. Metadata fields like `_routing` and
```

Comment:
I'd likely rewrite this whole section to make it more clear what stored fields are. Maybe it makes more sense to just make a page about them and reference it here like I did with `refresh.asciidoc`. Maybe this should come in a separate PR? I dunno.